src/coreclr/jit/gentree.h

```diff
@@ -4758,10 +4758,9 @@ struct GenTreeCall final : public GenTree
         // only used for CALLI unmanaged calls (CT_INDIRECT)
         GenTree* gtCallCookie;
         // gtInlineCandidateInfo is only used when inlining methods
-        InlineCandidateInfo*                  gtInlineCandidateInfo;
-        GuardedDevirtualizationCandidateInfo* gtGuardedDevirtualizationCandidateInfo;
-        ClassProfileCandidateInfo*            gtClassProfileCandidateInfo;
-        void*                                 gtStubCallStubAddr; // GTF_CALL_VIRT_STUB - these are never inlined
+        InlineCandidateInfo*       gtInlineCandidateInfo;
+        ClassProfileCandidateInfo* gtClassProfileCandidateInfo;
```

Comment:
I removed `gtGuardedDevirtualizationCandidateInfo` field because `GuardedDevirtualizationCandidateInfo` struct is smaller than `InlineCandidateInfo` (`InlineCandidateInfo` inherits `GuardedDevirtualizationCandidateInfo`) - so 

So e.g. `gtGuardedDevirtualizationCandidateInfo[1]` will return some junk.